jiant/preprocess.py

```diff
@@ -46,13 +46,15 @@
 ALL_SPLITS = ["train", "val", "test"]
 
 
-def _get_serialized_record_path(task_name, split, preproc_dir):
+def _get_serialized_record_path(task_name, tokenizer_name, split, preproc_dir):
     """Get the canonical path for a serialized task split."""
-    serialized_record_path = os.path.join(preproc_dir, "{:s}__{:s}_data".format(task_name, split))
+    serialized_record_path = os.path.join(
+        preproc_dir, "{:s}__{:s}_{:s}_data".format(task_name, tokenizer_name, split)
```

Comment:
Use double-underscores for the second and third separators as well? I don't think we rigorously enforce that names don't contain underscores elsewhere.